login/profile.go

```diff
@@ -53,13 +63,17 @@ func (keycloakUserProfile *KeycloakUserProfile) Update(accessToken string, keycl
 	if err != nil {
 		return errors.NewInternalError(err.Error())
 	}
-	req.Header.Add("Authorization", "Bearer "+accessToken)
-	req.Header.Add("Content-Type", "application/json")
+	userProfileClient.setHeader(req, "Authorization", "Bearer "+accessToken)
```

Comment:
Wanted to hide out details,but didnt help - removed :)
https://github.com/almighty/almighty-core/pull/1035/commits/06d35ba1c56c2a273462c534f238456669a45d24